command/action.go

```diff
@@ -121,7 +122,7 @@ EnvLoop:
 					continue EnvLoop
 				}
 			}
-			return "Error fetching variable " + envvar.string, envvar.error
+			return fmt.Errorf("Error fetching variable %v: %v"+envvar.string, envvar.error)
```

Comment:
I think this should be: `return fmt.Errorf("Error fetching variable %v: %v", envvar.string, envvar.error)`